scripts/check-each-commit.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+if [ "$1" = "" ]; then
```

Comment:
nit: it is recommended to use the `[[  ...  ]]` syntax to make sure a shell builtin (`[[`) is used and not an actual command (`[` is an alias of `test`) if the return code of a script is important to be evaluated.

Reply:
done!